features/old/services/subscriptions/bulk_operations/change_states.feature

```diff
@@ -20,7 +20,6 @@ Feature: Bulk operations
     Given current domain is the admin domain of provider "foo.3scale.localhost"
     Given I am logged in as provider "foo.3scale.localhost"
 
-  @wip
```

Comment:
A few things:

I would change how the steps are defined: first When, and then And, e.g. 
```
    When I am on the service contracts admin page
    And I follow "Account" within table
```

Also, to me "Do nothing" definition of the scenario is not very helpful, I'd say something like "Submit state change without selecting action"

Reply:
@mayorova Done